lib/usbmux/index.js

```diff
@@ -76,13 +93,16 @@ class Usbmux extends BaseServiceSocket {
    * @returns {?Object}
    */
   async readPairRecord (udid, timeout = 5000) {
-    const receivePromise = this._receivePlistPromise(timeout);
+    const { tag, receivePromise } = this._receivePlistPromise(timeout);
```

Comment:
should we update the docstring on this method?

Reply:
There is no docstring on it. It is a pretty simple and internal method. I don't think we need it here